project4/styles.css

```diff
@@ -0,0 +1,490 @@
+    font-weight: 700;
+    height: 70px;
+    margin: auto;
+    width: 200px;
+    padding-top: 15px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.button:active {
+    background: inherit;
+    border: 5px solid var(--color-lightgreen);
+    color: var(--color-lightgreen);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    height: 70px;
+    margin: auto;
+    width: 200px;
+    padding-top: 15px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.button-register {
+    color: var(--color-offwhite);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    margin: auto;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.button-register:hover {
+    color: var(--color-offwhite);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    margin: auto;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    text-align: center;
+    text-decoration: underline;
+    text-transform: uppercase;
+}
+
+.button-register:active {
+    color: var(--color-lightgreen);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    margin: auto;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    text-align: center;
+    text-decoration: underline;
+    text-transform: uppercase;
+}
+
+.conference-photo {
+    -webkit-filter: grayscale(100%);
+    -moz-filter: grayscale(100%);
+    -ms-filter: grayscale(100%);
+    -o-filter: grayscale(100%);
+    filter: grayscale(100%);
+    filter: gray;
+    opacity: 0.7;
+    height: 100%;
+    width: 100%;
+    padding-top: 20px;
+}
+
+.conference-photo:hover {
+    filter: grayscale(0);
+    opacity: 0.6;
+}
+
+
+.day2 {
+    padding-top: 20px;
+}
+
+.email {
+    text-align: center;
+}
+
+.jared-photo,
+.jake-photo,
+.kate-photo {
+    -webkit-filter: drop-shadow(8px 8px 10px black);
+    filter: drop-shadow(8px 8px 10px black);
+    opacity: 0.7;
+    width: 280px;
+    padding-top: 20px;
+}
+
+.jared-photo {
+    padding-top: 20px;
+}
+
+.jake-photo,
+.kate-photo {
+    padding-top: 40px;
+}
+
+.navigation-sst {
+    margin: auto;
+    padding-left: 0;
+}
+
+.navigation-name {
+    margin: auto;
+}
+
+.register {
+    text-align: center;
+}
+
+.speakers {
+    padding-top: 70px;
+}
+
+.schedule,
+.tickets {
+    color: var(--color-lightgreen);
+    padding-top: 70px;
+}
+
+.speakers-name {
+    padding-top: 20px;
+}
+
+.schedule-event,
+.speakers-name {
+    font-size: 20px;
+    font-weight: 600;
+    margin: 0;
+    text-transform: capitalize;
+}
+
+.schedule-time {
+    font-weight: 500;
+    margin: 5px 0;
+    text-transform: uppercase;
+    padding-top: 30px;
+    margin: 0;
```

Comment:
This rule is overwriting the rule you've written on line 318.

Note that this is one of the reasons I recommend alphabetizing your rules by property -- it's harder to miss these duplicates if they happen one right after the other; alphabetized, this ruleset would look like:

```
.schedule-time {
    font-weight: 500;
    margin: 5px 0;
    margin: 0;
    padding-top: 30px;
    text-transform: uppercase;
} 
```